### ISSUE TITLE
Chaper 2: Material UI example link is no longer valid

### ISSUE THREAD

**Issue by The-Corrupted** [async-labs/saas#210]
The chapter two link to the material ui nextjs-typescript example is no longer valid and will lead to a 404 page. The new url is:
https://github.com/mui/material-ui/tree/master/examples/material-ui-nextjs-ts

**Comment by wang2-lat** [comment]
Good catch. For future-proofing, consider linking to specific commit SHAs or release tags instead of `master` branch—prevents similar breakage when repos restructure. MUI's examples also live in their docs now (mui.com/material-ui/getting-started/example-projects/) which tends to be more stable than GitHub tree paths.
